acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -0,0 +1,163 @@
+
+    curl_authenticated('/puppet/v3/node/notme?environment=production')
+    assert_denied(/denied by rule 'puppetlabs node'/)
+
+    curl_unauthenticated('/puppet/v3/node/$(hostname -f)?environment=production')
+    assert_denied(/denied by rule 'puppetlabs node'/)
+  end
+
+  step 'report endpoint' do
+    curl_authenticated(report_query('$(hostname -f)'))
+    assert_allowed
+
+    curl_authenticated(report_query('notme'))
+    assert_denied(/denied by rule 'puppetlabs report'/)
+
+    curl_unauthenticated(report_query('$(hostname -f)'))
+    assert_denied(/denied by rule 'puppetlabs report'/)
+  end
+
+  step 'file_metadata endpoint' do
+    # We'd actually need to install a module in order to get back a 200,
+    # but we know that a 404 means we got past authorization
+    curl_authenticated('/puppet/v3/file_metadata/modules/foo?environment=production')
+    assert_allowed(404)
+
+    curl_unauthenticated('/puppet/v3/file_metadata/modules/foo?environment=production')
+    assert_denied(/denied by rule 'puppetlabs file'/)
+  end
+
+  step 'file_content endpoint' do
+    # We'd actually need to install a module in order to get back a 200,
+    # but we know that a 404 means we got past authorization
+    curl_authenticated('/puppet/v3/file_content/modules/foo?environment=production')
+    assert_allowed(404)
+
+    curl_unauthenticated('/puppet/v3/file_content/modules/foo?environment=production')
+    assert_denied(/denied by rule 'puppetlabs file'/)
+  end
+
+  step 'file_bucket_file endpoint' do
+    # We'd actually need to store a file in the filebucket in order to get
+    # back a 200, but we know that a 400 means we got past authorization
+    curl_authenticated('/puppet/v3/file_bucket_file/md5/123?environment=production')
+    assert_allowed(400)
+
+    curl_unauthenticated('/puppet/v3/file_bucket_file/md5/123?environment=production')
+    assert_denied(/denied by rule 'puppetlabs file'/)
+  end
+
+  step 'status endpoint' do
+    curl_authenticated('/puppet/v3/status/foo?environment=production')
+    assert_allowed
+
+    curl_unauthenticated('/puppet/v3/status/foo?environment=production')
+    assert_denied(/denied by rule 'puppetlabs status'/)
+  end
+
+  step 'certificate_revocation_list endpoint' do
+    curl_authenticated('/puppet-ca/v1/certificate_revocation_list/ca?environment=production')
+    assert_allowed
+
+    curl_unauthenticated('/puppet-ca/v1/certificate_revocation_list/ca?environment=production')
+    assert_denied(/denied by rule 'puppetlabs crl'/)
+  end
+
+  step 'certificate endpoint' do
+    curl_unauthenticated('/puppet-ca/v1/certificate/ca?environment=production')
+    assert_allowed
+  end
+
+  step 'certificate_request endpoint' do
+    # We'd actually need to store a CSR file on the server in order to get
+    # back a 200, but we know that a 404 means we got past authorization
+    curl_unauthenticated('/puppet-ca/v1/certificate_request/foo?environment=production')
+    assert_allowed(404)
```

Comment:
Might be good to include an example of a `put` here, too, to validate that the auth rule will allow it - since that's part of the rule definition.  Suppose if we did that that we could do a `get` right after that which would be more meaningful as well.